src/common/config.go

```diff
@@ -78,11 +78,14 @@ type FeaturesConfig struct {
 }
 
 type TraceConfig struct {
-	Cgroups bool `json:"cgroups"`
-	Memory  bool `json:"memory"`
-	Evictor bool `json:"evictor"`
-	Package bool `json:"package"`
-	Latency bool `json:"latency"`
+	Format        string `json:"format"`
+	Log_file_dir  string `json:"log_file_dir"`
```

Comment:
Do we need a separate directory for this, or just a path?  Is there more than one file?

Reply:
reuse the worker.out